xds/internal/xdsclient/client_new.go

```diff
@@ -89,13 +116,13 @@ type OptionsForTesting struct {
 	// unspecified, uses the default value used in non-test code.
 	WatchExpiryTimeout time.Duration
 
-	// AuthorityIdleTimeout is the timeout before idle authorities are deleted.
+	// IdleChannelExpiryTimeout is the timeout before idle channels are deleted.
```

Comment:
Optional: xDS Channels, and perhaps specify closed as well since

Reply:
Changed it to include the xds prefix. But deleting implies that the channel will be closed as well. So, did not add that explicitly.